js/sparkswap.js

```diff
@@ -318,8 +318,34 @@ module.exports = class sparkswap extends Exchange {
         };
     }
 
-    async fetchTrades () {
-        throw new ExchangeError ('Not Implemented');
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        const request = {
+            'market': symbol,
+        };
+        if (since) {
+            request['since'] = since;
+        }
+        if (limit) {
+            request['limit'] = limit;
+        }
+        let response = await this.privateGetV1Trades (request);
+        let trades = response['trades'];
+        let formattedTrades = [];
+        for (let i = 0; i < trades.length; i++) {
+            formattedTrades.push ({
+                'id': trades[i].id,
+                'timestamp': this.nanoToMillisecondTimestamp (trades[i].timestamp),
+                'datetime': this.nanoToMillisecondDatetime (trades[i].datetime),
+                'order': trades[i].order,
+                'symbol': trades[i].symbol,
```

Comment:
If you make the change I suggested in the broker: https://github.com/sparkswap/broker/pull/258/files#r222856588, then this will be trades[i].market or whatever